Query was executed successfully

{x,y} a
{0,0} 1
{0,1} 2
{0,2} 3
{1,0} 4
{1,1} 5
{1,2} 6
{2,0} 7
{2,1} 8
{2,2} 9

[An error expected at this place for the query "apply(mult_1, Z, 1.0 + 'a string')". And it failed with error code = scidb::SCIDB_SE_QPROC::SCIDB_LE_FUNCTION_NOT_FOUND. Expected error code = scidb::SCIDB_SE_QPROC::SCIDB_LE_FUNCTION_NOT_FOUND.]

Query was executed successfully

